tests/ccm.tests.ps1

```diff
@@ -0,0 +1,75 @@
+[CmdletBinding()]
+Param(
+    [Parameter(Mandatory)]
+    [String]
+    $Fqdn
+)
+
+Describe "Chocolatey Central Management Configuration" {
+    Context "Services" {
+        BeforeAll {
+            function Get-RemoteCertificate {
+                param(
+                    [Alias('CN')]
```

Comment:
Is this alias ever used? Personally I'd prefer not to have Aliases unless they're used, and I'd move towards just not using them at all inside of a script like this... (but of course if we're borrowing this function from somewhere else, keeping it stored there and then loading it in would be fine...